assets/css/style.css

```diff
@@ -718,6 +718,10 @@ textarea.form-control {
     font-size: 14px;
 }
 
```

Comment:
With reference to #62, added styling to the `highlight` function for improved UI of code blocks.